geomesa-spark/geomesa-spark-sql/src/test/scala/org/locationtech/geomesa/spark/SparkSQLTestUtils.scala

```diff
@@ -0,0 +1,55 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.spark
+
+import java.io.File
+import java.util.{Map => JMap}
+
+import com.vividsolutions.jts.geom.Coordinate
+import org.apache.spark.sql.SparkSession
+import org.geotools.data.simple.SimpleFeatureStore
+import org.geotools.data.{DataStore, DataUtilities}
+import org.geotools.geometry.jts.JTSFactoryFinder
+import org.joda.time.format.ISODateTimeFormat
+import org.locationtech.geomesa.features.ScalaSimpleFeature
+import org.locationtech.geomesa.utils.geotools.{GeneralShapefileIngest, SimpleFeatureTypes}
+
+import scala.collection.JavaConversions._
+
+object SparkSQLTestUtils {
+  def createSparkSession(): SparkSession = {
+    SparkSession.builder()
+      .appName("testSpark")
+      .config("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+      .config("spark.kryo.registrator", classOf[GeoMesaSparkKryoRegistrator].getName)
+      .config("spark.sql.crossJoin.enabled", "true")
+      .master("local[*]")
+      .getOrCreate()
+  }
+
+  def ingestChicago(ds: DataStore): Unit = {
```

Comment:
This function is duplicated in `GeoToolsSpatialRDDProviderTest`. Is there some way to avoid that?